### PR TITLE
Users update (backend)

### PR DESCRIPTION
**Pull request by EvanAntoni**
Another pull request into branch users_list.

### CODE REVIEW

**Review by MaxSukhovetskyi — APPROVED** [review]

File: easyrest/routes.py [hunk 1]
```diff
@@ -54,3 +54,4 @@ def includeme(config):
     config.add_route('order_status', 'order/{order_id:\d+}/status',
                      request_method=["GET", "PUT"])
     config.add_route('users_list', 'users/{role_id:\d+}', request_method="GET")
+    config.add_route('user_update', 'user/{user_id:\d+}', request_method="PUT")
```
Comment (MaxSukhovetskyi) [inline easyrest/routes.py:58]: You can aggregate CRUD logic in one route name (like 'user'), and have request_method=["PUT", "POST", "DELETE"] etc.

**Review by MaxSukhovetskyi — CHANGES_REQUESTED** [review]

File: easyrest/routes.py (hunk 1 above)
Comment (EvanAntoni) [reply easyrest/routes.py:58]: I thought about it. But chose to separate them. With this approach, it is more clear which parameter is expected and which action the request goes to + you can manage the URL more flexiblе (for example: `users/` for users list and `user/` for for a specific user).

Comment (MaxSukhovetskyi) [reply easyrest/routes.py:58]: Right now you have pull request "Users deletion (backend)" where you create new router name for deletion. Quote:
` config.add_route('user_delete', 'user/{user_id:\d+}', request_method="DELETE")`
Here you have:
`config.add_route('user_update', 'user/{user_id:\d+}', request_method="PUT")`
Both of them can be combined to something like:
`config.add_route('user', 'user/{user_id:\d+}', request_method=["PUT", "DELETE"])`

Comment (EvanAntoni) [reply easyrest/routes.py:58]: What's the difference?

**Review by MaxSukhovetskyi — APPROVED** [review]

**Review by RazSKg — APPROVED** [review]

**Pull request merged**